views/js/review/plugins/navigation/review-panel/scss/panel.scss

```diff
@@ -0,0 +1,273 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 Open Assessment Technologies SA ;
+ */
+@import "inc/bootstrap";
+
+$minPanelWidth: 250px;
+$maxPanelWidth: 250px;
```

Comment:
Why you need 2 same variables?

Reply:
Good catch. At the beginning the panel was floating between min and max size. After discussing with UX, we decided to fix the width. But I kept the logic in case of later change.